draft-ietf-tls-esni.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Encrypted ClientHello for TLS 1.3
-abbrev: TLS 1.3 Encrypted ClientHello
+title: Encrypted ClientHello for TLS
```

Comment:
If we are going to change the title, let's make it match the abbrev

"TLS Encrypted Client Hello"

Reply:
Works for me!